### ISSUE TITLE
Integration doesn't seem to handle Amazon if multiple shipments are part of the email

### ISSUE THREAD

**Issue by drjjr2** [moralmunky/Home-Assistant-Mail-And-Packages#722]
**Describe the bug**
I have an amazon shipment coming that contains items from two orders but do not see the package sensor change from zero. Looks like it has trouble with the line that says the shipment contains items from two orders.
 
**Environment (please complete the following information):**
 - OS: HassOS
 - x86
 - 2022.9.5
 - Hassio
 - Component version: 0.3.5-b14
**Logs**
```
2022-09-19 11:50:34.825 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Amazon emails found: 1
2022-09-19 11:50:34.952 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Email Multipart: True
2022-09-19 11:50:34.952 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Content Type: multipart/alternative
2022-09-19 11:50:34.952 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Amazon Subject: Your Amazon.com order #111-4572867-4776210 has shipped
2022-09-19 11:50:34.954 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Looking for: will arrive:
2022-09-19 11:50:34.955 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] First pass: Monday, September 19

This contains items from 2 orders
2022-09-19 11:50:34.955 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.959 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language it_IT. (unconverted data remains: 

This)
2022-09-19 11:50:34.960 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.963 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language it_IT.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:34.963 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.967 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pl_PL. (unconverted data remains: 

This)
2022-09-19 11:50:34.967 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.971 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pl_PL.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:34.971 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.975 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language de_DE. (unconverted data remains: 

This)
2022-09-19 11:50:34.976 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19
![Screenshot 2022-09-19 120354](https://user-images.githubusercontent.com/69729977/191073261-5f845e8e-d7f8-474e-9005-a61ad503d856.png)

This
2022-09-19 11:50:34.979 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language de_DE.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:34.979 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.982 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language es_ES. (unconverted data remains: 

This)
2022-09-19 11:50:34.983 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.986 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language es_ES.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:34.986 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.989 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pt_PT. (unconverted data remains: 

This)
2022-09-19 11:50:34.993 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:34.996 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pt_PT.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:34.997 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:35.000 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pt_BR. (unconverted data remains: 

This)
2022-09-19 11:50:35.000 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:35.002 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language pt_BR.UTF-8. (unconverted data remains: 

This)
2022-09-19 11:50:35.003 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Arrive Date: Monday, September 19

This
2022-09-19 11:50:35.009 DEBUG (SyncWorker_27) [custom_components.mail_and_packages.helpers] Invalid date format found for language . (unconverted data remains: 

This)
```

**Screenshots**

![Screenshot 2022-09-19 120354](https://user-images.githubusercontent.com/69729977/191073312-248f9eda-5fae-4f80-ba8d-7ac1774780d2.png)

**Additional context**
```
Subject: Your Amazon.com order #111-4572867-4776210 has shipped
MIME-Version: 1.0
Content-Type: multipart/alternative; boundary="----=_Part_9779460_241367992.1663566095446"
X-AMAZON-MAIL-RELAY-TYPE: notification
Bounces-to: 20220919054135f45202741ff8421da8ad5790a840p0na-C1B6W2W39A9SOV@bounces.amazon.com
X-AMAZON-METADATA: CA=C1B6W2W39A9SOV-CU=AKCXQSWZHDY9N
X-Original-MessageID: <urn.rtn.msg.20220919054135f45202741ff8421da8ad5790a840p0na@1663566095448.rtn-svc-na-back-c52xl-533b08aa.us-east-1.amazon.com>
Feedback-ID: 1.us-east-1.ZHcGJK6s+x+i9lRHKog4RW3tECwWIf1xzTYCZyUaiec=:AmazonSES
X-SES-Outgoing: 2022.09.19-54.240.13.135

------=_Part_9779460_241367992.1663566095446
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: 7bit

Amazon Shipping Confirmation
https://www.amazon.com?ie=UTF8&ref_=scr_home

____________________________________________________________________

Hi Don, your package will arrive:

Monday, September 19

This contains items from 2 orders

Track your package:
https://www.amazon.com/gp/your-account/ship-track?ie=UTF8&orderId=111-4572867-4776210&packageIndex=0&shipmentId=TxgMmqsh5&ref_=scr_pt_tp_t

JUST ASK
"Alexa, where's my stuff?"
https://www.amazon.com/?ref=sce_wms


On the way:
2 items
Order #111-4572867-4776210
See all order # information


An Amazon driver may contact you by text message or call you for help on the day of delivery.    
```

**Comment by firstof9** [comment]
Looks like a new format of the email.

**Comment by drjjr2** [comment]
Seems to be fixed in 0.3.5-b15. Thank you.
